packages/xrpl/package.json

```diff
@@ -80,6 +80,10 @@
     "url": "git@github.com:XRPLF/xrpl.js.git"
   },
   "readmeFilename": "README.md",
+  "keywords": [
+    "ripple-lib",
+    "ripple"
+  ],
```

Comment:
```suggestion
  "keywords": [
    "ripple-lib",
    "ripple",
    "xrp",
    "xrp ledger"
  ],
```

Reply:
I recommend adding these to all of the packages.

Reply:
I added all of them except ripple-lib because I think if people are searching for ripple-lib, they really are looking just for the `xrpl` package.